File: LATIHAN2/Latihan2_CPP/Product.cpp
#include <string>

using namespace std;

class Product
{
private:
	string idProduct;
	int price;

public:
	Product(){

	};
	Product(string idProduct, int pric)
	{
		this->idProduct = idProduct;
		this->price = price;
	};

	// setter and getter
	void setIdProduct(string idProduct)
	{
		this->idProduct = idProduct;
	};

	string getIdProduct()
	{
		return this->idProduct;
	};

	void setPrice(int price)
	{
		this->price = price;
	};

	int getPrice()
	{
		return this->price;
	}
};

File: LATIHAN2/Latihan2_Python/Product.py
class Product:
    # attribute class
    __idProduct = ""
    __price = 0

    # constructors
    def __init__(self, idProduct="", price=0):
        self.__idProduct = idProduct
        self.__price = price

    def setIdProduct(self, idProduct):
        self.__idProduct = idProduct

    def getIdProduct(self):
        return self.__idProduct

    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price
